playbooks/common/openshift-cluster/upgrades/v3_10/upgrade_control_plane.yml

```diff
@@ -45,6 +46,59 @@
   - set_fact:
       pre_upgrade_complete: True
 
+- import_playbook: label_nodes.yml
```

Comment:
We shouldn't need to label nodes here.  Everything should have been labeled properly during 3.9 upgrades and during 3.9 / 3.10 installs.

Reply:
No, 3.10 changes a default to make infra nodes correct.  3.9 doesn’t have the correct labels.